src/main/java/org/folio/auth/authtokenmodule/tokens/RefreshToken.java

```diff
@@ -10,12 +9,33 @@
  * Refresh tokens are provided to obtain a new access token.
  * @see AccessToken
  */
+
 public class RefreshToken extends Token {
   /**
    * A string representation of the type of this token.
    */
   public static final String type = "refresh";
 
+  public UUID getId() {
+    return UUID.fromString(claims.getString("jti"));
+  }
+
+  public UUID getUserId() {
+    return UUID.fromString(claims.getString("user_id"));
+  }
+
+  public long getExpiresAt() {
+    return claims.getLong("exp");
+  }
+
+  /**
+   * Should only be used by tests.
+   * @param to The epoch seconds time stamp to the iat claim to.
```

Comment:
```suggestion
   * @param to The epoch seconds time stamp to set the iat claim to.
```

Reply:
Thanks. It's also to `set the exp claim` rather than `iat`.